bundles/org.openhab.binding.anel/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,268 @@
+		<channel-groups>
+			<channel-group id="prop" typeId="propertiesGroup"/>
+
+			<channel-group id="r1" typeId="relayGroup"/>
+			<channel-group id="r2" typeId="relayGroup"/>
+			<channel-group id="r3" typeId="relayGroup"/>
+		</channel-groups>
+
+		<!--
+			<properties>
+			<property name="vendor">ANEL Elektronik AG</property>
+			<property name="modelId">NET-PwrCtrl HOME</property>
+			</properties>
+		-->
+		<representation-property>macAddress</representation-property>
+
+		<config-description-ref uri="thing-type:anel:config"/>
+	</thing-type>
+
+	<thing-type id="simple-firmware">
+		<label>Anel NET-PwrCtrl PRO / POWER</label>
+		<description>Anel device with 8 controllable outlets without IO ports.</description>
+
+		<channel-groups>
+			<channel-group id="prop" typeId="propertiesGroup"/>
+
+			<!-- Example channel ID: anel:simple-firmware:mydevice:r1#state -->
+			<channel-group id="r1" typeId="relayGroup"/>
+			<channel-group id="r2" typeId="relayGroup"/>
+			<channel-group id="r3" typeId="relayGroup"/>
+			<channel-group id="r4" typeId="relayGroup"/>
+			<channel-group id="r5" typeId="relayGroup"/>
+			<channel-group id="r6" typeId="relayGroup"/>
+			<channel-group id="r7" typeId="relayGroup"/>
+			<channel-group id="r8" typeId="relayGroup"/>
+		</channel-groups>
+
+		<!--
+			<properties>
+			<property name="vendor">ANEL Elektronik AG</property>
+			<property name="modelId">NET-PwrCtrl PRO / POWER</property>
+			</properties>
+		-->
+		<representation-property>macAddress</representation-property>
+
+		<config-description-ref uri="thing-type:anel:config"/>
+	</thing-type>
+
+	<thing-type id="advanced-firmware">
+		<label>Anel NET-PwrCtrl ADV / IO / HUT</label>
+		<description>Anel device with 8 controllable outlets / relays and possibly 8 IO ports.</description>
+
+		<channel-groups>
+			<channel-group id="prop" typeId="propertiesGroup"/>
+
+			<channel-group id="r1" typeId="relayGroup"/>
+			<channel-group id="r2" typeId="relayGroup"/>
+			<channel-group id="r3" typeId="relayGroup"/>
+			<channel-group id="r4" typeId="relayGroup"/>
+			<channel-group id="r5" typeId="relayGroup"/>
+			<channel-group id="r6" typeId="relayGroup"/>
+			<channel-group id="r7" typeId="relayGroup"/>
+			<channel-group id="r8" typeId="relayGroup"/>
+
+			<channel-group id="io1" typeId="ioGroup"/>
+			<channel-group id="io2" typeId="ioGroup"/>
+			<channel-group id="io3" typeId="ioGroup"/>
+			<channel-group id="io4" typeId="ioGroup"/>
+			<channel-group id="io5" typeId="ioGroup"/>
+			<channel-group id="io6" typeId="ioGroup"/>
+			<channel-group id="io7" typeId="ioGroup"/>
+			<channel-group id="io8" typeId="ioGroup"/>
+
+			<!-- Example channel ID: anel:advanced-firmware:mydevice:sensor#humidity -->
+			<channel-group id="sensor" typeId="sensorGroup"/>
+
+			<!-- power measurement is not yet supported by any device but already covered by the UDP protocol:
+				<channel-group id="power" typeId="powerGroup"/>
+			-->
```

Comment:
Dead code should be avoided. You could remove the power measurement related code and description and save it with `git stash` for further releases.